liquidprompt

```diff
@@ -1740,6 +1742,14 @@ _lp_set_prompt()
         LP_VENV=
     fi
 
+    # Display the current NVM selection, if available
+    if [[ "$LP_ENABLE_NVM,${NVM_BIN-}" = 1,?* ]]; then
+        LP_NVM="${NVM_BIN##*node/v}"
+        LP_NVM=" [${LP_COLOR_NVM}node-${LP_NVM%/bin}${NO_COL}]"
```

Comment:
Looking at this again, why remove `node/v` to add back `node-`? This means a env name of `node/v1.2.3` is displayed as `node-1.2.3`, which could be confusing when people try to switch between versions. Why not simply display the actual env var?

Reply:
The answer is that the env var contains the entire path to the `bin` directory, like:
```
/home/user/.nvm/versions/node/v18.1.0/bin
```